Add additional licenses to "Upload" selection

josephsnyder opened the pull request:
Add a few other licenses to the selection on the Upload page to
select from when finalizing a submission.

Add GPL, LGPL, and BSD as additional selections and update the display
on each submission page.

josephsnyder committed:
Add additional licenses to "Upload" selection

Add a few other licenses to the selection on the Upload page to
select from when finalizing a submission.

Add GPL, LGPL, and BSD as additional selections and update the display
on each submission page.

BetsyMcPhail commented:
The license isn't showing up in the 'pending approval' emails. e.g.

```
Hello,

A new submission needs your approval.

Title: svsd.

Authors: sd s.

Abstract: sd

License:

Agreed to Contributed Software Attribution Policy: No

Result of Survey script: http://localhost/midas/journal/survey?id=314

To approve this submission please login in and go to here.

Generated by OSEHRA Technical Journal.
```

josephsnyder committed:
Fix Approval email content with new licenses

Fix the information that goes into the "Send for Approval" email to
have the new license choices and use a new function in the resourcedao
to get the license string
